packages/next/src/index.ts

```diff
@@ -2595,18 +2609,39 @@ async function getServerlessPages(params: {
   pagesDir: string;
   entryPath: string;
   outputDirectory: string;
+  appPathRoutesManifest?: Record<string, string>;
 }) {
-  const [pages, middlewareManifest] = await Promise.all([
+  const [pages, appPaths, middlewareManifest] = await Promise.all([
     glob('**/!(_middleware).js', params.pagesDir),
+    params.appPathRoutesManifest
+      ? glob('**/page.js', path.join(params.pagesDir, '../app'))
+      : Promise.resolve({}),
     getMiddlewareManifest(params.entryPath, params.outputDirectory),
   ]);
 
+  const normalizedAppPaths: typeof appPaths = {};
+
+  if (params.appPathRoutesManifest) {
+    for (const entry of Object.keys(params.appPathRoutesManifest || {})) {
```

Comment:
Nit pick: use `Object.entries()` here to get key and value at the same time

Reply:
Updated in https://github.com/vercel/vercel/pull/8098/commits/8953047b143e9134b30a4b9f12acaabada795af0